packages/threat-composer/src/components/generic/WindowExporter/index.tsx

```diff
@@ -93,9 +93,7 @@ const WindowExporter: FC<PropsWithChildren<{}>> = ({ children }) => {
     window.threatcomposer.renameWorkspace = renameWorkspace;
   }, [renameWorkspace]);
 
-  useEffect(() => {
-    window.threatcomposer.stringifyWorkspaceData = stringifyWorkspaceData;
-  }, []);
+  window.threatcomposer.stringifyWorkspaceData = stringifyWorkspaceData;
```

Comment:
We do not need this. It has been defined above in ```window.threatcomposer = {
  stringifyWorkspaceData,```

Reply:
Removed in commit